jenkins/job-configs/kubernetes-jenkins-pull/charts-pull.yaml

```diff
@@ -0,0 +1,100 @@
+- job:
+    name: 'charts-pull-test-e2e'
+    concurrent: true
+    properties:
+        - build-discarder:
+            days-to-keep: 7
+        - github:
+            url: 'https://github.com/kubernetes/charts'
+        - throttle:
+            max-total: 4
+            max-per-node: 2
+            option: project
+        - raw:
```

Comment:
yeah, these lines are basically how we healthcheck nodes, since Jenkins doesn't really do that for us natively. I'm fine leaving them in here. (we should probably refactor this into a custom property that we use in all of our jobs to avoid duplication, but that's out of scope for this PR.)